Samples/GTK/gtk-gui/MainWindow.cs

```diff
@@ -0,0 +1,28 @@
+
+// This file has been generated by the GUI designer. Do not modify.
+
+public partial class MainWindow
+{
+	private global::Gtk.Image barcodeImage;
+
+	protected virtual void Build()
+	{
+		global::Stetic.Gui.Initialize(this);
+		// Widget MainWindow
+		this.Name = "MainWindow";
+		this.Title = global::Mono.Unix.Catalog.GetString("MainWindow");
+		this.WindowPosition = ((global::Gtk.WindowPosition)(4));
+		// Container child MainWindow.Gtk.Container+ContainerChild
+		this.barcodeImage = new global::Gtk.Image();
+		this.barcodeImage.Name = "barcodeImage";
+		this.Add(this.barcodeImage);
+		if ((this.Child != null))
+		{
+			this.Child.ShowAll();
+		}
+		this.DefaultWidth = 400;
+		this.DefaultHeight = 300;
+		this.Show();
+		this.DeleteEvent += new global::Gtk.DeleteEventHandler(this.OnDeleteEvent);
+	}
+}
```

Comment:
nit: use space instead of tabs in this file

Reply:
I think we can change it because it's autogenerated file 😞

Reply:
mkay